db/migrate/20170331041221_create_ctas.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCtas < ActiveRecord::Migration[5.0]
+  def change
+    create_table :ctas, id: :uuid do |t|
+      t.string :title
+      t.text :description
+      t.string :website
+      t.boolean :free, default: true
+      t.datetime :start_at
+      t.datetime :end_at
+      t.integer :action_type
+      t.uuid    :location_id
+      t.uuid    :contact_id
+      t.uuid    :call_script_id
+
+      t.timestamps
```

Comment:
I think the criteria we've got in this PR are sufficient for determining whether a CTA is unique.  Let me know if you disagree.